client/src/pages/Home.jsx

```diff
@@ -1,31 +1,99 @@
+import { useState } from "react";
 import TopPlanet from "../components/TopPlanet";
 import HomeCards from "../components/HomeCards";
 import Slogan from "../components/Slogan";
+import ReservationModule from "../components/ReservationModule";
+import { GalactapediaProvider } from "../Contexts/GalactapediaContext";
 import "../styles/app.scss";
 
 function Home() {
+  const [thumbnailUrl1, setThumbnailUrl1] = useState(" ");
+  const [titleUrl1, setTitleUrl1] = useState(" ");
+
+  const [thumbnailUrl2, setThumbnailUrl2] = useState(" ");
+  const [titleUrl2, setTitleUrl2] = useState(" ");
+
+  const [thumbnailUrl3, setThumbnailUrl3] = useState(" ");
+  const [titleUrl3, setTitleUrl3] = useState(" ");
+
+  const [thumbnailUrl4, setThumbnailUrl4] = useState(" ");
+  const [titleUrl4, setTitleUrl4] = useState(" ");
+
+  const [thumbnailUrl5, setThumbnailUrl5] = useState(" ");
+  const [titleUrl5, setTitleUrl5] = useState(" ");
+
+  const [thumbnailUrl6, setThumbnailUrl6] = useState(" ");
+  const [titleUrl6, setTitleUrl6] = useState(" ");
```

Comment:
smol function could be better here ;)

Reply:
or not i didn't seen the following lines... idk how a function would help here, but it must be possible